### PR TITLE
builder - templates - overview 

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR generalizes the article uilders/build/templates/overview/, removing polkadot as the only security provider

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `tanssi-mkdocs` to update redirects

### CODE REVIEW

File: builders/build/templates/overview.md [hunk 1]
```diff
@@ -1,58 +1,48 @@
 ---
 title: Templates
-description: Tanssi includes templates to kick-start the development of an appchain, one for a Substrate-oriented runtime and another featuring full EVM (Ethereum) support.
+description: Tanssi includes templates to kick-start the development of an appchain offering options such as a basic one and another featuring full EVM (Ethereum) support.
```
Comment (albertov19) [inline builders/build/templates/overview.md:3]:
```suggestion
description: Tanssi includes templates to kick-start the development of an appchain with full EVM (Ethereum) support and minimal engineering overhead.
```

Comment (albertov19) [reply builders/build/templates/overview.md:3]: I think it was discussed that we just mention the EVM template btw (considering the previous PRs we merged)

File: builders/build/templates/overview.md [hunk 2]
```diff
@@ -1,58 +1,48 @@
...
 ---
 
 # Templates Overview {: #templates-overview }
 
 ## Introduction {: #introduction }
 
-Appchains deployed through Tanssi are essentially parachains within the Polkadot ecosystem that are capable of interacting with the relay chain and other parachains. They also contain the necessary functionalities to support the Tanssi protocol.
+Appchains deployed through Tanssi are fully customizable blockchains benefiting from a shared set of sequencers and the security of a provider of their choice ([Symbiotic](https://symbiotic.fi/){target=\_blank}, for example). They also need to implement the necessary functionalities and configurations to support the Tanssi protocol, which the templates presented in this article already do, making it easier for developers.
```
Comment (albertov19) [inline builders/build/templates/overview.md:10]:
```suggestion
Appchains deployed through Tanssi are fully customizable blockchains benefiting from a shared set of sequencers and the security of a provider of their choice (for example, [Symbiotic](https://symbiotic.fi/){target=\_blank} for Ethereum). They also need to implement the necessary functionalities and configurations to support the Tanssi protocol, which the templates presented in this article already do, making it easier for developers.
```

File: builders/build/templates/overview.md [hunk 3]
```diff
@@ -1,58 +1,48 @@
...
 
 As presented in the [Included Templates](/learn/tanssi/included-templates/#baseline-appchain-template){target=\_blank} from the Learn section, Tanssi already provides two templates to jumpstart the development process:
 
-- **[Baseline Tanssi appchain template](/learn/tanssi/included-templates/#baseline-appchain-template){target=\_blank}** - a template that provides the basic platform to start adding custom logic in a Substrate-based Tanssi appchain
+- **[Baseline Tanssi appchain template](/learn/tanssi/included-templates/#baseline-appchain-template){target=\_blank}** - a template that provides the basic platform to start adding custom logic
```
Comment (albertov19) [inline builders/build/templates/overview.md:14]: Maybe we remove? Unsure

File: builders/build/templates/overview.md [hunk 4]
```diff
@@ -1,58 +1,48 @@
...
+- **Runtime Upgrades** - a runtime upgrade in a Tanssi appchain must be informed to the security provider's operators to allow them to check on the blocks produced by the sequencers of the Tanssi appchains
 
 ## Included Modules {: #included-modules }
 
-Besides the necessary modules to support the operation of the Tanssi appchain as part of the broader Polkadot ecosystem and the modules that enable the Tanssi protocol and its block production mechanism, many other modules provide functional behavior that the users can interact with. 
+Besides the necessary modules to support the operation of a Tanssi appchain, many other modules provide functional behavior that the users can interact with. 
 
 These are some of the functional modules exposing a behavior to the users that are included in the templates and ready to use:
 
 - **[pallet_balances](https://paritytech.github.io/substrate/master/pallet_balances/index.html){target=\_blank}** - the Balances pallet provides functions for handling accounts and balances for the Tanssi appchain native currency
```
Comment (albertov19) [inline builders/build/templates/overview.md:42]: Call them pallets still?

**Review by albertov19 — CHANGES_REQUESTED** [review]
Love how things are panning out. Left some minor comments. Unsure if we will use still word pallets, maybe we just rename that as Module and not call it the official way

File: builders/build/templates/overview.md (hunk 4 above)
Comment (augustocollado) [reply builders/build/templates/overview.md:42]: Nope, I missed those!

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**